composer.json

```
@@ -27,7 +27,7 @@
         "rmccue/requests" : "^2.0",
         "pomo/pomo" : "^1.4",
         "geoip2/geoip2" : "^2.10",
-        "aura/sql": "^3.0",
+        "aura/sql": "^6.0.0",
```

Let's keep only two digits for the version

---

The Caret Version Range actually says: `>=6.0.0 <7.0.0`

---

Indeed, but streamlining to two digits somewhat eases the maintenance.

```suggestion
        "aura/sql": "^6.0",
```